src/app/Http/Controllers/Operations/FetchOperation.php

```diff
@@ -57,9 +57,12 @@ private function fetch($arg)
         $model_instance = new $config['model'];
         // set configuration defaults
         $config['paginate'] = isset($config['paginate']) ? $config['paginate'] : 10;
-        $config['searchableAttributes'] = $config['searchableAttributes'] ?? $model_instance->identifiableAttribute();
+        $config['searchable_attributes'] = $config['searchable_attributes'] ?? $model_instance->identifiableAttribute();
```

Comment:
Should we make this name a little bit more human ? Like `search_attr` or `search_in` ?